main_routers/workshop_router.py

```
@@ -2027,20 +2028,100 @@ async def prepare_workshop_upload(request: Request):
-            return JSONResponse({
-                "success": False,
-                "error": f"模型目录不存在: {model_name}"
-            }, status_code=404)
-        
-        # 复制整个模型目录到临时目录
-        model_dest_dir = os.path.join(temp_item_dir, model_name)
-        shutil.copytree(model_dir, model_dest_dir, dirs_exist_ok=True)
-        logger.info(f"模型文件已复制到临时目录: {model_dest_dir}")
+        # 2. 根据模型类型查找并复制模型文件
+        if model_type in ('vrm', 'mmd'):
+            # VRM/MMD 模型：model_name 是文件路径如 /user_vrm/model.vrm 或 /user_mmd/folder/model.pmx
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)